praneershrest/Capsule
# feature/outfit-history

To test:
1. Change the mobile_navigation.xml file **line 10** to `android:name="com.example.capsule.ui.outfitHistory.OutfitHistory"`
2. Change the mobile_navigation.xml file **line 12** to `tools:layout="@layout/fragment_outfit_history"`

Preview of page display
<img width="348" alt="Screenshot 2022-11-30 at 5 24 36 AM" src="https://user-images.githubusercontent.com/51378040/204807771-e86b83a2-152a-43d8-814e-c0693cac5570.png">

TODO:
1. Navigational logic to this fragment
2. Pictures should be cropped to a square size so the display is more aesthetic and clean
